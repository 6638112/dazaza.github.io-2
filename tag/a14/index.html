<!doctype html><html lang="zh-hans"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>#a14</title><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"><link rel="stylesheet" href="/img/css.css?random="><link data-rh="true" rel="icon" href="/img/favicon.ico"/><script data-ad-client="ca-pub-6067137220025946" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><script type="text/javascript" src="https://platform-api.sharethis.com/js/sharethis.js#property=5effb96910009800120b8d4d&product=inline-share-buttons" async="async"></script>
<script>var _hmt = _hmt || [];(function() {var hm = document.createElement("script");hm.src = "https://hm.baidu.com/hm.js?03c1a0f31299b4a2fbb83c34d6beaac9";var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(hm, s);})();</script></head><body><div id="my_header"><div class="container"><nav class="navbar navbar-expand-lg"><a class="navbar-brand" href="/"><img alt="diglog" src="/img/logo.v1.gif" class="rounded-sm"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNavAltMarkup"><div class="navbar-nav"></div></div></nav></div></div><div class="container"><div id="my_content"><h1>#a14</h1><div class="row"><div class="col-lg-8 col-12"><div id="list"><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1031603.html">Apple的A14封装134M晶体管/mm²</a></div><div class="item_title_en"><a target="_blank" href="https://semianalysis.com/apples-a14-packs-134-million-transistors-mm2-but-falls-far-short-of-tsmcs-density-claims/">Apple’s A14 Packs 134M Transistors/mm²</a><span>(semianalysis.com)</span></div><span class="my_story_list_date">2020-10-28 4:18</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1031603.html"><img src="http://img2.diglog.com/img/2020/10/thumb_6a18472e3f63b15d2e78ded9d16b01b4.png" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">我们在ICmaster的朋友们已经深入研究了苹果A14仿生的包装。芯片尺寸已经被揭开，达到88mm2。尽管塞入了118亿个晶体管，但由于采用了台积电的5 nm工艺节点，芯片尺寸非常小。
进步的进程并非一帆风顺。苹果的芯片历来在其处理器中达到了工艺节点理论密度的90%以上。这一代人之所以脱颖而出，是因为他们在很大程度上......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/a14/">#a14</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/sram/">#sram</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1025152.html">A14处理器不情愿的首次亮相</a></div><div class="item_title_en"><a target="_blank" href="https://sixcolors.com/post/2020/09/the-reluctant-debut-of-the-a14-processor/">The reluctant debut of the A14 processor</a><span>(sixcolors.com)</span></div><span class="my_story_list_date">2020-9-20 17:47</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1025152.html"><img src="http://img2.diglog.com/img/2020/9/thumb_958cd297961edb4a467584b61c2a414d.jpeg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">周二，苹果发布了一系列新产品，包括两款新的苹果手表，两款新的iPad，一项新的健身订阅服务，以及它的一系列服务。
几乎可以肯定的是，这不是该公司计划的首次亮相。苹果喜欢在今年最大的活动-新款iPhone的发布会上宣扬过去一年在芯片设计方面的创新。九年前，A系列芯片在iPhone1之外首次亮相。
但事实证明，2020年......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/a14/">#a14</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/debut/">#debut</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/苹果/">#苹果</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1024365.html">苹果发布5纳米A14 SoC-微不足道的升级，还是只是不那么耗电？</a></div><div class="item_title_en"><a target="_blank" href="https://www.anandtech.com/show/16088/apple-announces-5nm-a14-soc-meagre-upgrades-or-less-power-hungry">Apple Announces 5nm A14 SoC – Meagre Upgrades, or Just Less Power Hungry?</a><span>(www.anandtech.com)</span></div><span class="my_story_list_date">2020-9-17 3:42</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1024365.html"><img src="http://img2.diglog.com/img/2020/9/thumb_5440cef2fcfc299ddd8f78fb1f6f096a.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">在今天发布的新iPad和Watch设备中，苹果发布了新的A14 SoC芯片，这是一个新闻。苹果公司的最新一代硅设计值得注意的是，这是业界第一个在5 nm工艺节点上制造的商业芯片，这标志着这是预计将在半导体领域大幅突破极限的新一代设计中的第一个。
苹果今年披露的活动有点令人困惑，因为该公司正在将新的A14指标与A12进......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/发布/">#发布</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/苹果/">#苹果</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/soc/">#soc</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/a14/">#a14</a></button></div></div></div></div><div class="col-lg-4 col-0"><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/美国/">#美国</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/程序/">#程序</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/rust/">#rust</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/数据/">#数据</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/warning/">#warning</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/代码/">#代码</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/ai/">#ai</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/linux/">#linux</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/学习/">#学习</a></button></div><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/web2.0/">#web2.0</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/创意/">#创意</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/摄影/">#摄影</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/图片/">#图片</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/视频/">#视频</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/手机/">#手机</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/广告/">#广告</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/iphone/">#iphone</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/网站/">#网站</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/免费/">#免费</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/下载/">#下载</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/微软/">#微软</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/firefox/">#firefox</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/苹果/">#苹果</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/blog/">#blog</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/音乐/">#音乐</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/博客/">#博客</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/wordpress/">#wordpress</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/恶搞/">#恶搞</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/艺术/">#艺术</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/qq/">#qq</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/web/">#web</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/工具/">#工具</a></button></div></div></div></div><div id="my_footer"><div class=""><a href="/tags/">tags</a> <a href="/users/">users</a></div>&copy;2012-2021 diglog.com </div></div></body></html>