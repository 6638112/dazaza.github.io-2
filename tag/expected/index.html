<!doctype html><html lang="zh-hans"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>#expected</title><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"><link rel="stylesheet" href="/img/css.css?random="><link data-rh="true" rel="icon" href="/img/favicon.ico"/><script data-ad-client="ca-pub-6067137220025946" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><script type="text/javascript" src="https://platform-api.sharethis.com/js/sharethis.js#property=5effb96910009800120b8d4d&product=inline-share-buttons" async="async"></script>
<script>var _hmt = _hmt || [];(function() {var hm = document.createElement("script");hm.src = "https://hm.baidu.com/hm.js?03c1a0f31299b4a2fbb83c34d6beaac9";var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(hm, s);})();</script></head><body><div id="my_header"><div class="container"><nav class="navbar navbar-expand-lg"><a class="navbar-brand" href="/"><img alt="diglog" src="/img/logo.v1.gif" class="rounded-sm"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNavAltMarkup"><div class="navbar-nav"></div></div></nav></div></div><div class="container"><div id="my_content"><h1>#expected</h1><div class="row"><div class="col-lg-8 col-12"><div id="list"><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1046034.html">资料来源：风险投资人已与Clubhouse接触，希望以约$ 1B的估值进行下一轮投资，较上一轮增加10倍； A16Z有望领先 </a></div><div class="item_title_en"><a target="_blank" href="https://www.theinformation.com/articles/clubhouse-gets-investment-interest-at-1-billion-valuation">Sources: VCs have approached Clubhouse about investing in its next round at ~$1B valuation, a 10X increase from the previous round; a16z is expected to lead</a><span>(www.theinformation.com)</span></div><span class="my_story_list_date">2021-1-24 3:41</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1046034.html"><img src="http://img2.diglog.com/img/2021/1/thumb_b64e4de167039d566b74c5e04ada2fa6.jpeg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">投资社交音频应用程序Clubhouse的竞赛再次开始。据四位知情人士透露，风险投资公司近几周已与这家初创公司的创始人接触，希望以约10亿美元的估值领导其下一轮潜在融资。
 一位知情人士说，投资者还试图以隐含的估值10亿美元从现有股东手中购买Clubhouse的股票。在经过激烈竞争的投资回合之后，这个数字比八个月前的应......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/风险投资/">#风险投资</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/接触/">#接触</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1038540.html">DoorDash预计将以每股$ 102的价格对IPO定价，价格为$ 39B </a></div><div class="item_title_en"><a target="_blank" href="https://www.wsj.com/articles/doordash-poised-to-tap-torrid-ipo-market-11607442406">DoorDash Is Expected to Price IPO at Around $102 a Share at $39B Valuation</a><span>(www.wsj.com)</span></div><span class="my_story_list_date">2020-12-9 19:31</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1038540.html"><img src="http://img2.diglog.com/img/2020/12/thumb_5b20b26cfd09bf904a0c80b6ca34928e.jpeg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">知情人士说，DoorDash Inc.在首次公开募股中的发行价约为每股102美元，这标志着这家外卖公司和整个IPO市场的强劲表现。
  按照完全摊薄后的股票数量计算，该价格约为390亿美元，其中包括此次发行所筹集的33.7亿美元的收益。此前，这家旧金山公司提高了上周最初的预期，此前的目标价为每股90美元至95美元。
......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/doordash/">#doordash</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1035436.html">英国预计将柴油禁令再提前5年</a></div><div class="item_title_en"><a target="_blank" href="https://thenextweb.com/shift/2020/11/16/uk-expected-to-bring-diesel-ban-forward-by-another-5-years/">UK expected to bring diesel ban forward by another 5 years</a><span>(thenextweb.com)</span></div><span class="my_story_list_date">2020-11-16 21:28</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1035436.html"><img src="http://img2.diglog.com/img/2020/11/thumb_ec12eaa19cf20196032c12de04b1be65.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">英国首相预计将在本周宣布一些措施，届时英国对汽油和柴油乘用车的禁令将再延长五年。
据英国《金融时报》报道，鲍里斯·约翰逊(Boris Johnson)将公布一系列新的绿色倡议，其中包括将化石燃料禁令从2035年提前到2030年。
不过，这项禁令将不会像以前那样具有极权主义色彩。预计部长们将保留2035年作为混合动力汽......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/预计/">#预计</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/禁令/">#禁令</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1035204.html">英国预计从2030年起禁止销售新的汽油和柴油汽车</a></div><div class="item_title_en"><a target="_blank" href="https://www.theguardian.com/environment/2020/nov/14/uk-expected-to-ban-sale-of-new-petrol-and-diesel-cars-from-2030">UK expected to ban sale of new petrol and diesel cars from 2030</a><span>(www.theguardian.com)</span></div><span class="my_story_list_date">2020-11-15 7:55</span><div class="my_story_list_item_desc">据悉，鲍里斯·约翰逊(Boris Johnson)计划在十年内禁止销售新的汽油和柴油车，有报道称，这一禁令将提前五年实施。
在此之前，英国首相在2月份将截止日期从2040年改为2035年。
根据英国《金融时报》的一篇报道，约翰逊预计将于下周在一系列新的环境政策中宣布这一措施。该报道将这一消息归因于行业和白厅消息来源。......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/柴油/">#柴油</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/英国/">#英国</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1030346.html">/*您不需要理解这一点*/</a></div><div class="item_title_en"><a target="_blank" href="https://community.cadence.com/cadence_blogs_8/b/breakfast-bytes/posts/memorial-day">/* You Are Not Expected to Understand This */</a><span>(community.cadence.com)</span></div><span class="my_story_list_date">2020-10-22 8:51</span><div class="my_story_list_item_desc">这篇帖子的标题，你是不会理解的，这是有史以来最著名的评论之一。它位于Unix的V6版本中，Unix是Linux、Android和iOS的母版操作系统。UNIX是由贝尔实验室的丹尼斯·里奇和肯·汤普森于20世纪70年代初开发的。当我说注释时，我使用的是编译器忽略的源代码注释的技术意义上的这个词。
Unix的V6版本特别......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/需要/">#需要</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/进程/">#进程</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1030282.html">冠状病毒每年造成的死亡人数比预期多出近30万人。</a></div><div class="item_title_en"><a target="_blank" href="https://www.washingtonpost.com/health/coronavirus-excess-deaths/2020/10/20/1e1d77c6-12e1-11eb-ba42-ec6a580836ed_story.html">Coronavirus has caused nearly 300k more deaths than expected in typical year</a><span>(www.washingtonpost.com)</span></div><span class="my_story_list_date">2020-10-21 19:9</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1030282.html"><img src="http://img2.diglog.com/img/2020/10/thumb_c514170f2a221bae594dded50b390114.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">“华盛顿邮报”免费提供了有关冠状病毒的这一重要信息。要获得更多关于冠状病毒大流行的免费报道，请订阅我们的冠状病毒更新时事通讯，所有故事都可以免费阅读。
美国疾病控制和预防中心周二报道，冠状病毒大流行已导致美国死亡人数比通常一年的预期多29.9万人，其中三分之二死于新冠肺炎，其余死于其他原因。
疾控中心表示，正如之前的......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/病毒/">#病毒</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/冠状病毒/">#冠状病毒</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/死亡/">#死亡</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1023853.html">Red Ventures表示，已同意以5亿美元从ViacomCBS手中收购CNET Media Group，交易预计将于2020年第四季度完成</a></div><div class="item_title_en"><a target="_blank" href="https://variety.com/2020/digital/news/cnet-viacomcbs-sells-red-ventures-1234768810/">Red Ventures says it has agreed to buy CNET Media Group from ViacomCBS for $500M in a deal expected to close in Q4 2020</a><span>(variety.com)</span></div><span class="my_story_list_date">2020-9-14 20:33</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1023853.html"><img src="http://img2.diglog.com/img/2020/9/thumb_02dea5e3ffeca66f55db1d875d4e63a0.png" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">正在与数字营销公司达成5亿美元的交易，出售媒体集团(Media Group)。数字营销公司周一宣布了这笔交易。
根据协议，CNET媒体集团执行副总裁和总经理马克·拉金，以及该部门的高级管理人员，将在收购后继续留在Red Ventures。CNet的出售预计将在2020年第四季度完成。CNET传媒集团有980名员工，“......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/2020/">#2020</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/交易/">#交易</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/ventures/">#ventures</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/cnet/">#cnet</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1015536.html">美国国家航空航天局以比任何人预期的更快的速度拥抱了人类飞行的再利用</a></div><div class="item_title_en"><a target="_blank" href="https://arstechnica.com/science/2020/07/even-before-crew-dragon-returns-from-space-nasa-endorses-reuse-for-humans/">More quickly than anyone expected, NASA embraces reuse for human flights</a><span>(arstechnica.com)</span></div><span class="my_story_list_date">2020-8-1 7:29</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1015536.html"><img src="http://img.diglog.com/img/2020/8/thumb_8bfcadcc437dd5795264c4995a4d9953.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">如果天气允许，SpaceX的乘员“龙”号飞船将于周日在墨西哥湾溅落。天气预报员正在密切关注飓风伊萨亚斯造成的情况，但希望任务能在佛罗里达狭长地带发现平静的海面和微风。
与在太平洋返回地球的阿波罗任务不同，NASA和SpaceX选择了佛罗里达半岛附近的溅落目标。他们这样做的主要原因是为了让宇航员在完成太空飞行后更快地返......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/美国/">#美国</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/nasa/">#nasa</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/拥抱/">#拥抱</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/spacex/">#spacex</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1007441.html">特朗普预计将暂停H-1B和其他签证，直到年底</a></div><div class="item_title_en"><a target="_blank" href="https://www.npr.org/2020/06/20/881245867/trump-expected-to-suspend-h-1b-other-visas-until-end-of-year">Trump Expected to Suspend H-1B, Other Visas Until End of Year</a><span>(www.npr.org)</span></div><span class="my_story_list_date">2020-6-21 2:32</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1007441.html"><img src="http://img.diglog.com/img/2020/6/thumb_bb7e12c70b8c6f8190c244a8e0d22a98.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">据熟悉该计划的多名消息人士透露，特朗普总统预计最快将于周六签署一项命令，暂停H-1B、L-1和其他临时工作签证，直至年底。
这项新命令-预计会有广泛的例外-出台之际，政府正在继续努力应对由于冠状病毒大流行而导致的美国工人的高失业率，并启动经济复苏。
该命令将针对H-1B签证和L-1签证，H-1B签证是为某些熟练工人(......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/直到/">#直到</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/签证/">#签证</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1000793.html">“瘟疫公司”的开发者从未想过现实会像它的视频游戏一样。</a></div><div class="item_title_en"><a target="_blank" href="https://www.vice.com/en_us/article/bv8g8z/plague-inc-developer-never-expected-reality-to-look-like-its-video-game">'Plague, Inc.’ developer never expected reality to look like its video game</a><span>(www.vice.com)</span></div><span class="my_story_list_date">2020-5-6 1:53</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1000793.html"><img src="http://img.diglog.com/img/2020/5/thumb_cf05bee9112af8dc340f8fd1208a9269.jpeg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">大流行使他们的游戏重新登上排行榜榜首，现在他们正在进行更新，在那里你可以结束一场大流行，而不是导致它。</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/开发/">#开发</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/expected/">#expected</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/排行榜/">#排行榜</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/pandemic/">#pandemic</a></button></div></div></div></div><div class="col-lg-4 col-0"><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/美国/">#美国</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/程序/">#程序</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/rust/">#rust</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/数据/">#数据</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/warning/">#warning</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/代码/">#代码</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/ai/">#ai</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/linux/">#linux</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/学习/">#学习</a></button></div><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/web2.0/">#web2.0</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/创意/">#创意</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/摄影/">#摄影</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/图片/">#图片</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/视频/">#视频</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/手机/">#手机</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/广告/">#广告</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/iphone/">#iphone</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/网站/">#网站</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/免费/">#免费</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/下载/">#下载</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/微软/">#微软</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/firefox/">#firefox</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/苹果/">#苹果</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/blog/">#blog</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/音乐/">#音乐</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/博客/">#博客</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/wordpress/">#wordpress</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/恶搞/">#恶搞</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/艺术/">#艺术</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/qq/">#qq</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/web/">#web</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/工具/">#工具</a></button></div></div></div></div><div id="my_footer"><div class=""><a href="/tags/">tags</a> <a href="/users/">users</a></div>&copy;2012-2021 diglog.com </div></div></body></html>